abstract_commands.xml

```diff
@@ -5,8 +5,26 @@
         is accessible, then \Fsize matching the register's actual size must be
         supported.
 
+        \begin{steps}{Perform the following sequence of operations:}
+        \item Halt the hart, if \Fprehalt is set.
+        \item Execute the Program Buffer, if \Fpreexec is set.
+        \item Copy data from the specified register into {\tt data}, if \Fwrite is clear.
+        \item Copy data from {\tt data} into the specified register, if \Fwrite is set.
+        \item Execute the Program Buffer, if \Fpostexec is set.
+        \item Resume the hart, if \Fpostresume is set.
+        \end{steps}
+
         <field name="0" bits="31:24" />
-        <field name="0" bits="23:22" />
+        <field name="prehalt" bits="23">
+            When 1, halt the hart before performing the rest of the command.
```

Comment:
Ah, this describes something different than what I understood. So actually what this does is very quickly, under the direction of the debugger, snatch something from the hart.

So this doesn't cover another possible case of interest, where a hart hits a breakpoint and quickly dumps a value to the debugger, or jumps to some other monitor code? Is that case covered only by usual breakpoint interrupt code?